lessons/pydata/api/index.ipynb

```diff
@@ -0,0 +1,3684 @@
+{
```

Comment:
<p>Asi bychom to neměli zveřejňovat. Můžeme to dát k dispozici při hodině přes Slack</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='105'/>